### ISSUE TITLE
No Softmax Function in Actor Network in rl PPO demo code

### ISSUE THREAD

**Issue by ziaoang** [keras-team/keras-io#550]
In the file examples/rl/ppo_cartpole.py, line 234-235
`logits = mlp(observation_input, list(hidden_sizes) + [num_actions], tf.tanh, None)`
`actor = keras.Model(inputs=observation_input, outputs=logits)`
We can see the actor network doesn't have a output_activation function which should be Softmax function

**Comment by ziaoang** [comment]
wrong understand
